### PR TITLE
docs: add ts as const example in zod enums

### PR DESCRIPTION
**Pull request by recallwei**
## What I do in this PR:
- add space between English letters and Chinese characters for readability
- add `as const` example (missing in zh doc) in `zod enums`

### CODE REVIEW

File: deno/lib/README.md [hunk 1]
```diff
@@ -57,8 +57,8 @@
 - [Coercion for primitives](#coercion-for-primitives)
 - [Literals](#literals)
 - [Strings](#strings)
-  - [Datetime](#datetime-validation)
-  - [IP](#ip-address-validation)
+  - [Datetime](#iso-datetimes)
+  - [IP](#ip-addresses)
```
Comment (scotttrinh) [inline deno/lib/README.md:61]: Can we move all of the changes in this deno README to a separate PR if they're relevant?

**Review by scotttrinh — CHANGES_REQUESTED** [review]

File: deno/lib/README.md (hunk 1 above)
Comment (recallwei) [reply deno/lib/README.md:61]: I have not modified this file, the possible reason is that the build automatically generates.

Comment (recallwei) [reply deno/lib/README.md:61]: I re-committed again and removed the changed code. But there is no diff.

Comment (recallwei) [reply deno/lib/README.md:61]: @scotttrinh

Comment (scotttrinh) [reply deno/lib/README.md:61]: @recallwei 

I think the issue is that you need to merge `origin:master` into your branch and try again.

Comment (recallwei) [reply deno/lib/README.md:61]: @scotttrinh Ok, I synced the master branch.

**Comment by colinhacks** [comment]
Thank you!

**Pull request merged**